docker-files/zeeguu-web/Dockerfile

```diff
@@ -0,0 +1,109 @@
+FROM python:3.6
+
+# Build arguments with default values, overwrite at build-time using
+# "--build-arg <varname>=<value>"
+ARG DEFAULT_ZEEGUU_API="https://zeeguu.unibe.ch/api"
+# The public IP where the zeeguu API can be reached on
+# It will be used in the static JS
+# ZEEGUU_API__EXTERNAL is accessed from the static JS client code.
+ARG ZEEGUU_API__EXTERNAL=http://127.0.0.1:9001
```

Comment:
this is only used if an external is not provided, right? 
maybe make sure to mention this here too.